detection-rules/impersonation_benefits_enrollment.yml

```
@@ -0,0 +1,116 @@
+name: "Benefits Enrollment Impersonation"
+description: "Detects messages about benefit enrollment periods and healthcare selections from external senders that contain urgent language or requests for action. Excludes legitimate HR communications, marketing mailers, and trusted sender domains with valid authentication."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and sender.email.domain.domain not in $org_domains
+  and length(body.current_thread.text) < 2500
```

```suggestion
  and length(body.current_thread.text) < 2500 or length(body.current_thread.text) is null
```
handle null values in body.current_thread.text